src/config.rs

```diff
@@ -32,7 +32,7 @@ pub struct ClientConfig {
     pub no_prompt: bool
 }
 
-/// declares the `ApiKey` type which implements the Header trait
+// declares the `ApiKey` type which implements the Header trait
```

Comment:
The `///` is for [doc comments](https://doc.rust-lang.org/rust-by-example/meta/doc.html). I don't think this should have been changed.